Linux Script/configure-linux.sh

```diff
@@ -215,18 +218,23 @@ checkIfSupportedOS()
 	case "$LINUX_DIST_IN_LOWER_CASE" in
 		*"ubuntu"* )
 		echo "INFO: Operating system is Ubuntu."
+		PKG_MGR="apt-get"
 		;;
-		*"redhat"* )
+		*"red"* )
 		echo "INFO: Operating system is Red Hat."
+		PKG_MGR="yum"
```

Comment:
I'm surprised you don't just check for the presence of the young command, instead of trying to hardcode it based on the distribution